buildpack/runtime_components/storage.py

```diff
@@ -65,17 +75,63 @@ def _get_s3_specific_config(vcap_services, m2ee):
     v2_auth = os.getenv("S3_USE_V2_AUTH", v2_auth).lower() == "true"
     sse = os.getenv("S3_USE_SSE", "").lower() == "true"
 
-    if not (access_key and secret and bucket):
+    if access_key and secret and bucket:
+        logging.info("S3 config detected, activating external file store")
+        config = {
+            "com.mendix.core.StorageService": "com.mendix.storage.s3",
+            "com.mendix.storage.s3.AccessKeyId": access_key,
+            "com.mendix.storage.s3.SecretAccessKey": secret,
+            "com.mendix.storage.s3.BucketName": bucket,
+        }
+    elif tvm_endpoint and tvm_username and tvm_password and bucket:
+        if m2ee.config.get_runtime_version() < 9.2:
+            logging.info(
+                "S3 config detected, fetching IAM credentials from TVM"
+            )
+            response = requests.get(
+                "https://%s/v1/getcredentials" % tvm_endpoint,
+                headers={
+                    "User-Agent": "Mendix Runtime %s"
+                    % util.get_buildpack_version()
+                },
+                auth=(tvm_username, tvm_password),
+            )
+            if not response.ok:
+                logging.error("Failed to get IAM credential from TVM")
+                return None
+            result = response.json()
+
+            if "AccessKeyID" in result:
+                access_key = result["AccessKeyID"]
+            if "SecretAccessKey" in result:
+                secret = result["SecretAccessKey"]
+
+            if access_key and secret:
+                config = {
+                    "com.mendix.core.StorageService": "com.mendix.storage.s3",
+                    "com.mendix.storage.s3.AccessKeyId": access_key,
+                    "com.mendix.storage.s3.SecretAccessKey": secret,
+                    "com.mendix.storage.s3.BucketName": bucket,
+                }
+            else:
+                logging.error(
+                    "Failed to get AccessKeyID or SecretAccessKey from TVM"
+                )
+                return None
+        else:
+            logging.info("S3 config detected, activating external file store")
```

Comment:
Could the text be altered a little, like "S3 TVM config detected, activating external file store"? Then we can see the difference between using the TVM and regular IAM credentials (on line 79).